src/Sylius/Bundle/PaymentBundle/Resources/config/app/config.yml

```diff
@@ -7,13 +7,3 @@ jms_serializer:
             sylius-payment:
                 namespace_prefix: "Sylius\\Component\\Payment"
                 path: "@SyliusPaymentBundle/Resources/config/serializer"
-
-sylius_payum:
```

Comment:
unfortunately, it is BC-Break. It should stay and be removed in 2.0

Reply:
I think it's not released yet and don't exists in 1.12 as this a very recent change related to api-bundle validation.

Are we allowed to make BC between 1.13-alpha.1 and 1.13-alpha.2?

Reply:
I would vote yes, since that's IMO the goal of an alpha.